src/assets/lang/en.json

```diff
@@ -24,8 +24,13 @@
   "overValue": "Over {startValue}",
 
   "entityParagraph": "This {variableLink} applies to the {entityLink}",
+  "entityRolesParagraph": "This {entityLink} defines the following roles:",
   "variableText": "variable",
-  "entityText": "entities",
+  "entityText": "entity",
+  "entityRoles": "Roles",
+  "roleDescription": "Description",
+  "roleMaxPersons": "Maximal number of persons for this role",
```

Comment:
> I find "for this role" a little heavy. Doesn't the table structure indicate that this is the max for the role?

Reply:
I agree, it's heavy but, imho, we need to set the unit of max (max what).
`Max persons`? 🤔

Reply:
`Maximal number of persons` ?